### PR TITLE
Added recipe and LICENSE file for enum-compat package

### PR DESCRIPTION
**Pull request by trendelkampschroer**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/enum_compat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - enum
+
+about:
+  home: https://github.com/jstasiak/enum-compat
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: This is a virtual package, its whole purpose is to install enum34 on Python older than 3.4. On Python 3.4+ it’s a no-op.
+  # The remaining entries in this section are optional, but recommended.
```
Comment (jan-janssen) [inline recipes/enum_compat/meta.yaml]: You can remove this line.

Comment (trendelkampschroer) [reply recipes/enum_compat/meta.yaml]: Done.

**Comment by trendelkampschroer** [comment]
@jan-janssen Thanks a lot for your review. If this looks good to you, then I'd be glad if you could close this. We rely on this package to become available in order to update another feedstock which depends on  enum-compat.

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Pull request merged**